extensions/elasticsearch-rest-client-common/deployment/src/main/java/io/quarkus/elasticsearch/restclient/common/deployment/DevServicesElasticsearchProcessor.java

```diff
@@ -0,0 +1,219 @@
+package io.quarkus.elasticsearch.restclient.common.deployment;
+
+import java.time.Duration;
+import java.util.*;
```

Comment:
Avoid star imports

Reply:
I thought we had automated format checks... why did this slip through the cracks? :/

Reply:
Ok there are dozens of start imports throughout the codebase, clearly the rule is not that strict. I'll try to look into this (and into why the formatter doesn't prevent start imports) later, but for now I'll go ahead and merge.